src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/integration/wiremock/HmppsAuthMockServer.kt

```diff
@@ -1,6 +1,7 @@
 package uk.gov.justice.digital.hmpps.incidentreporting.integration.wiremock
 
 import com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import com.github.tomakehurst.wiremock.client.WireMock.get
```

Comment:
what does this do? doesn't ktlint complain about an unused import?

Reply:
fixed